src/main/java/uk/gov/companieshouse/api/accounts/service/impl/AbstractServiceImpl.java

```diff
@@ -40,11 +40,8 @@ public T save(T rest, String companyAccountId) throws NoSuchAlgorithmException {
 
     @Override
     public U findById(String id) {
-        Optional<U> optional = (Optional<U>) mongoRepository.findById(id);
-        if (optional.isPresent()) {
-            return optional.get();
-        }
-        return null;
+        Optional<U> optional = mongoRepository.findById(id);
```

Comment:
Pretty sure this can be cut even more?
`return mongoRepository.findById(id).orElse(null);`